src/pages/events.js

```diff
@@ -119,6 +92,33 @@ export default function Events({
             ))}
           </Carousel>
         </Section>
+        <Section divider={"border-primary"}>
+            <h1 className="h1">Calendar</h1>
+            <section className="w-full space-y-5 md:space-y-[1.875rem] markdown layout-narrow">
+            <p className="h2">
+             Schedule for official Urbit events. In addition to in-person, we regularly hold online events where you can hang out, learn, and get involved. Most of these are in the Urbit Hacker House, a shared virtual office space.
+              </p>
+            </section>
+            <Link
+            className="btn bg-primary hover:bg-secondary text-surface body-lg w-min"
+            href="https://calendar.google.com/calendar/ical/c_13647438d00ef31237be88b19de24de30aeb2609657c80cfb6b22350941c61dd%40group.calendar.google.com/public/basic.ics"
+            >
+            Subscribe to Calendar
+            </Link>
```

Comment:
When I look at this page in Chrome or Firefox it looks like the two buttons are joined together. I'd add some spacing between them with `marginRight`, tweak the exact value as you see fit.

```jsx
<Link
  style={{ marginRight: '20px' }}
  className="btn bg-primary hover:bg-secondary text-surface body-lg w-min"
  href="https://calendar.google.com/calendar/ical/c_13647438d00ef31237be88b19de24de30aeb2609657c80cfb6b22350941c61dd%40group.calendar.google.com/public/basic.ics"
>
  Subscribe to Calendar
</Link>
```

Reply:
Thank you, I'll put this in.